### ISSUE TITLE
Alerting Stack

### ISSUE THREAD

**Issue by flomotlik** [theserverlessway/aws-baseline#13]
**Is your feature request related to a problem? Please describe.**
When sending Budget or in the future Auditing Alerts a standardised SNS Topic to send Alerts through should be implemented. Support for AWS Chatbot could be another way to integrate this.

Instead of having to set your Emails or other notification options in different stacks they should be centralised

**Describe the solution you'd like**
SNS Topics for various Levels (Security, Auditing, Info, Warn, ERROR, ...) that get exported into the CF Exports.

**Comment by mwarkentin** [comment]
Chatbot just hooks into SNS so I don’t think it’s either/or. 😊

**Comment by flomotlik** [comment]
Yup definitely both. In the end it will probably be an alert topic exported into CF that the Chatbot hooks into, potentially even as another stack
